### DISCUSSION TITLE
Should there be separate method for original SIMPOL.1?

### DISCUSSION THREAD

**Discussion by Aariq** [Meredith-Lab/volcalc#50] [Q&A] [answered]
Should `volcalc` have an option to use the original Pankow & Asher SIMPOL coefficients in addition to the version modified in Meredith et al. (e.g. that adds coefs for amides, sulfates, etc.).  It seems like users might want the option to use the original.

**Comment by KristinaRiemer** [comment]
That's a good question!

**Reply by Aariq** [reply]
Another follow-up question is whether `calc_vol()` should accept *multiple* methods.  That would probably involve appending the method name to output columns like `log_P_simpol1`, `volatililty_simpol1`, `log_P_meredith`, `volatility_meredith`, etc.

**Comment by Aariq** [comment, marked as answer]
Discussed at meeting and yes, there should be options for `"meredith"` and `"simpol1"`
